--- aewan-0.9.3/Makefile.in~	2004-08-17 19:51:30.000000000 +0000
+++ aewan-0.9.3/Makefile.in	2004-10-19 04:30:25.195123664 +0000
@@ -73,11 +73,11 @@
 	rm -f configure
 
 install: all
-	mkdir -p $(bindir)
-	cp $(execs) $(bindir)
-	mkdir -p $(mandir)/man1 $(mandir)/man5
-	cp man/man1/*.1 $(mandir)/man1
-	cp man/man5/*.5 $(mandir)/man5
+	mkdir -p $(DESTDIR)$(bindir)
+	cp $(execs) $(DESTDIR)$(bindir)
+	mkdir -p $(DESTDIR)$(mandir)/man1 $(DESTDIR)$(mandir)/man5
+	cp man/man1/*.1 $(DESTDIR)$(mandir)/man1
+	cp man/man5/*.5 $(DESTDIR)$(mandir)/man5
 
 -include depends
 
